### ISSUE TITLE
[Feature Request] Use a different 'prefix' on redis instead of default prefix 'waterline'

### ISSUE THREAD

**Issue by visitsb** [balderdashy/sails-redis#76]
I can't seem to spot any easier way to change the default redis prefix `waterline` to a prefix of my choice. Is this possible?

**Comment by Ryanc1256** [comment]
@visitsb you can't atm, but why would you want to anyway?

**Comment by visitsb** [comment]
Thanks. 

Nothing urgent right now, but before using waterline, I was using [nohm](http://maritz.github.io/nohm/#prefix) as an ORM for `redis` which allowed me to set my own prefixes. 

This allowed me to share the same `redis` instance for different applications. I thought that would be useful for `waterline` too as I'd not prefer to use same prefix for multiple applications sharing same `redis` instance. 

On the other hand, sails supports `prefix` for storing sessions on a redis instance, so from a consistency angle I'd have assumed that would be the case for `sails-redis` too. (This might be a different question, but when `sails` is set to use `redis` for session, does it use `sails-redis` under the covers?)

**Comment by Ryanc1256** [comment]
hmm, interesting it's worth looking into anyway, i'll give it a look through see if i can add something in like that.

And if i remember correctly it doesn't, it just use's it's own system based off which it uses (this)[https://github.com/tj/connect-redis]

which is used for the session store for express which sails uses

**Comment by visitsb** [comment]
Thanks. 

If there is a possibility to use the _same_ adapter for `redis` under the covers (_sails-redis, connect-redis et al_), 
I think it makes life easier to look at adapters _consistently_ no matter if they're used for sessions, or data storage.

Appreciate the great work!

**Comment by Ryanc1256** [comment]
@visitsb Yeah totally agree with you, would make things like this more consistent under the covers for sails too..

I'll give it a talk over with the contributors of sails

**Comment by sailsbot** [comment]
Thanks for posting, @visitsb.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
